scene/register_scene_types.cpp

```diff
@@ -767,7 +767,11 @@ void register_scene_types() {
 
 void unregister_scene_types() {
 
-	clear_default_theme();
+	Theme::set_project_default(NULL);
+	Theme::set_default(NULL);
```

Comment:
`clear_default_theme()` was already doing what those 4 extra lines now do, no? Why replace it? It's now unused.

Reply:
Ooh, let me change that...

Reply:
Okay, should be better now!